src/graph/transform/to_bipartite.h

```diff
@@ -19,7 +19,7 @@ namespace transform {
 template<DLDeviceType XPU, typename IdType>
 std::tuple<HeteroGraphPtr, std::vector<IdArray>, std::vector<IdArray>>
 ToBlock(HeteroGraphPtr graph, const std::vector<IdArray> &rhs_nodes,
-        bool include_rhs_in_lhs);
+        bool include_rhs_in_lhs, std::vector<IdArray> lhs_nodes);
```

Comment:
Seems that `lhs_nodes` should be mutable and is always the same object as the second element in the returned tuple?  If that's the case I will no longer return `lhs_nodes` but instead make `lhs_nodes` an output parameter (`std::vector<IdArray> &`).

Reply:
Switched to an in/out parameter in 
add691b. However, using a pointer as the linter doesn't like non-const references.